pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/readers/ImmutableDictionaryTypeConversionTest.java

```diff
@@ -59,6 +60,13 @@ public class ImmutableDictionaryTypeConversionTest {
   private static final int STRING_LENGTH = 6;
   private static final int BYTES_LENGTH = STRING_LENGTH / 2;
 
+  private static byte[] toUTF8Bytes(String stringValue) {
+    return stringValue.getBytes(StandardCharsets.UTF_8);
+  }
+  private static ByteArray toUTF8ByeArray(String stringValue) {
+    return new ByteArray(toUTF8Bytes(stringValue));
+  }
+
```

Comment:
Suggest removing them and just inline it (there is also a typo in the method name :P)

Reply:
Done